_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,38 @@
+
+More specifically, thie year we want to make HackIllinois even more _exciting_. We are putting as much emphasis as we can on making this event more than 36 hours of coding; simply learning about open source is not enough at HackIllinois 2019. It will be the immersive experience of teamwork, guidance, and excitement that hackers have come to expect out of our staff. We will run interactive workshops, present lively mini-events, and host top industry talent for HackIllinois 2019. The Open Source experience will be a cut above that of the past: exciting to both the OSS gurus and those who have not yet explored this region of the technology world.
+
+# Something Borrowed
+
+We are always open to feedback and new ideas. If you have participated in our event or any hackathon, conference, get-together, Urbana Sweetcorn Festival, or ANYTHING where you found a fun or engaging project, activity, or meal, do not hesitate to suggest it to us. We are still in the brainstorming phases for the event and love to hear from you all, so reach out to us! Working together and getting advice from informed individuals is our style. #(OUR HASHTAG HERE)
+
+There are some pilot ideas we will be exploring over the following months as planning continues, so let us know on contact@hackillinois.org if you have suggestions, feedback, or just want to share your experiences with us. We're always happy to hear from you! We will continue to make more announcements on our blog, right here, and on our social media.
+
+Likewise, we will be attending conferences and hackathons alike to get a good sense of what is currently being developed in the world around us. If you see somebody in a HackIllinois staff shirt, please engage with us! We genuinely enjoy having conversations with our attendees, and it may be a much easier forum to discuss and get ideas across than over an email. Additionally, if you are (or will soon become) an Illinois student who'd like to get involved in this process and join our team, we will make a call for staff applications in the fall, so stay tuned :).
+
+# Something Blue
+
+Our team is currently hard at work incorporating the color blue into the design of HackIllinois 2019.
```

Comment:
Good teaser but we should wrap up the post more clearly.

Reply:
if this post is published before we publish landing page, we should go back and link our landing page here since ocean/deep sea == blue and good place to start advertising

Reply:
I put the landing page design in the header of the post, and linked the landing page at the bottom. If we release them both close together I figure it'd work well